scripts/gqm_gen/gqm.md

```diff
@@ -0,0 +1,18 @@
+
+> gqmgen@1.0.0 start
+
+```mermaid
```

Comment:
Is this the output?  Maybe put in a comment to note that it is autogenerated.

Reply:
Yes, that's the output.

It shouldn't actually be checked in. :)

I'll definitely add a README.md.

I'll get to buttoning this up soon, maybe in today's mob session.